panic on cacheACL type assertion in example

os:
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

ovn-nbctl (Open vSwitch) 2.9.5
DB Schema 5.10.0

ovn-sbctl (Open vSwitch) 2.9.5
DB Schema 1.15.0


#### how to reproduce:

go run examples/main.go

#### error message
```bash
{b7c93ac6-d263-47dc-bd9b-d1e7c10da7d5 test  map[] [12:34:56:78:90 10.10.10.1] []   map[]}
panic: interface conversion: interface {} is nil, not libovsdb.OvsSet

goroutine 1 [running]:
github.com/ebay/go-ovn.(*ovndb).rowToACL(0xc0000b2120, 0xc00001c4e0, 0x24, 0x0)
        /home/max/GolandProjects/go-ovn/acl.go:280 +0xa31
github.com/ebay/go-ovn.(*ovndb).aclListImp(0xc0000b2120, 0x74cc9c, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/max/GolandProjects/go-ovn/acl.go:322 +0x36c
github.com/ebay/go-ovn.(*ovndb).ACLList(0xc0000b2120, 0x74cc9c, 0x3, 0x1, 0x0, 0x0, 0x51, 0x74cdf7)
        /home/max/GolandProjects/go-ovn/client.go:426 +0x3f
main.main()
        /home/max/GolandProjects/go-ovn/examples/main.go:70 +0x803

--- a/acl.go
+++ b/acl.go
@@ -277,7 +277,7 @@ func (odbi *ovndb) rowToACL(uuid string) *ACL {
 	case string:
 		severity = cacheACL.Fields["severity"].(string)
 	case libovsdb.OvsSet:
-		for _, a := range cacheACL.Fields["meter"].(libovsdb.OvsSet).GoSet {
+		for _, a := range cacheACL.Fields["severity"].(libovsdb.OvsSet).GoSet {
 			severity = a.(string)
 		}
 	default:

